packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -147,8 +150,27 @@ export default class DatePicker extends React.Component {
     super(props)
     this.state = this.calcInitialState()
     this.handleOnPosition = this.handleOnPosition.bind(this);
+    this.handleKeydown = this.handleKeydown.bind(this);
     this.datePickerContainer = React.createRef();
     this.datePickerHookShotContainer = React.createRef();
+    this.datePickerOverlayContainer = React.createRef();
+    this.escapeKey = KeyCode.KEY_ESCAPE;
```

Comment:
Do we need to store this on the instance?

Reply:
Updated in this commit: https://github.com/cerner/terra-framework/pull/608/commits/148cfbfb7fa7abcf5cd1ae14bf89bf2be7496b9d